Parsers/InfobloxNIOS/InfobloxNIOS.txt

```diff
@@ -27,21 +27,14 @@
 //
 let RawData = Syslog 
     | where Computer in ("datasource") and Facility == "local6"
```

Comment:
It seems Facility is set as daemon, if its right in SampleData then we have to change in the query from local6 to daemon.

Reply:
This line is used as an example, I had initially intended it to be commented out, leaving the user to modify it according to how their environment is set up. In the instructions (#2) the user is to change this filter to match their data feed. Let me know if we should change how we present this for the other data connectors as well. @preetikr

Reply:
@ashwin-patil that was the only sample we were able to obtain at the moment.